Now, Vinegar has a GUI. (#77)
vinegarhq/vinegarhq.github.io: src/Home/index.md

- Vinegar is a transparent, minimal, configurable, and fast bootstrapper for Roblox Player and Roblox Studio.
- The software and documentation are both open source, and can be accessed [on GitHub](https://github.com/vinegarhq).

> - There is no <abbr title="Graphical User Interface">GUI</abbr> for Vinegar at this time, but there's a loading screen.

Reviewer: ```suggestion
- There is no graphical configuration interface for Vinegar at this time.
```
